docs/content/latest/api/ysql/the-sql-language/statements/ddl_truncate.md

```diff
@@ -2,7 +2,7 @@
 title: TRUNCATE
 headerTitle: TRUNCATE
 linkTitle: TRUNCATE
-description:  Use the TRUNCATE statement to clear all rows in a table.
+description:  Use the TRUNCATE statement to remove all rows from one or several tables.
```

Comment:
That single-word change makes things a lot clearer! (The "from one or several" is helpful, too, but "remove" is a big improvement.)

Reply:
Yep... I don't know who wrote this. I tried to stay away from re-writing every page that I had to touch. But I made this change. You were simply saying "well-done", yes? I don't think that you're asking for further change.